cooltools/cli/saddle.py

```diff
@@ -286,7 +289,7 @@ def saddle(
     else:
         max_diag = -1
     if clr_weight_name:
-        track = api.saddle.mask_bad_bins(
```

Comment:
Tests ran so probably not! ?

Reply:
yes, you're right - only used in CLI before digitization of the `track` ...

But now it makes me wonder if this step should be better incorporated into one of the user-facing functions ?
right now it is done in CLI outside of any user-facing functions `digitize` or `saddle` - it does not fit into `digitize` now, because `digitize` does not know about the `cooler` (now), but by the time we get to `api.saddle` it is too late ...

Does it "deserve" to be brought into `digitize` by making it depend on the `cooler` and/or some `bad_bin_mask` (optionally of course)?

Is masking "bad" bins important for saddle ?:
- masking bad bins in the track seems important and (?)could affect the result of the saddle(?) via outliers , maybe ...
- maybe should check with a track defined for every bin (e.g. GC-content) and test with and without masking bad bins from cooler

Reply:
should we make this Q a separate issue and proceed with the split ?

Reply:
Yes let's make this a separate issue-- there may even be one about saddles that this can be added to